.github/workflows/publish-to-live-pypi.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Python 🐍 distributions 📦 to pypi
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to pypi
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/djangocms-snippet
```

Comment:
This needs to match what the pypi project is (or will be when creating these for new projects)